package.json

```diff
@@ -101,6 +98,7 @@
   },
   "dependencies": {
     "ember-cli-babel": "^7.1.2",
+    "ember-decorators-polyfill": "^1.0.3",
```

Comment:
This addon doesnt need a hard dependency on `ember-decorators-polyfill` since we just export a decorator function with no dependencies.

This can be moved into devDeps.